src/sentry/event_manager.py

```diff
@@ -106,19 +107,22 @@ def plugin_is_regression(group, event):
 
 
 def has_pending_commit_resolution(group):
-    return (
+    """
+    Checks that the most recent commit that fixes a group has had a chance to release
+    """
+    recent_group_link = (
         GroupLink.objects.filter(
             group_id=group.id,
             linked_type=GroupLink.LinkedType.commit,
             relationship=GroupLink.Relationship.resolves,
         )
-        .extra(
-            where=[
-                "NOT EXISTS(SELECT 1 FROM sentry_releasecommit where commit_id = sentry_grouplink.linked_id)"
-            ]
-        )
-        .exists()
+        .order_by("-datetime")
```

Comment:
Is `datetime` correct here? I feel like this date just represents the date the link was created in our system. Would the date on the `Commit` table be more accurate?

Reply:
I think we want them in the order that the activity page displays them in. So that it feels similar to "resolved in next release" in the UI